packages/radio-player/package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "@internetarchive/radio-player",
+  "version": "0.0.0",
+  "description": "A Radio Player LitElement.",
+  "license": "AGPL-3.0-only",
+  "main": "index.js",
+  "module": "index.js",
+  "types": "index.d.ts",
+  "files": [
+    "lib/**/*",
+    "!lib/demo",
+    "index.d.ts",
+    "index.js"
+  ],
+  "scripts": {
+    "prepare": "npm run build",
```

Comment:
Suggestion: Use `yarn` to stay in parity with previous iaux components. We also decided yarn was our main package manager, perhaps we need to revisit

Reply:
Done